remoteworkitem/remoteworkitem_test.go

```diff
@@ -1,5 +1,3 @@
-// +build unit
-
```

Comment:
I was already taking care of this in https://github.com/almighty/almighty-core/pull/227  :)